pkg/service/dbaas/service.go

```diff
@@ -237,3 +252,27 @@ func hasAggregatedInstances(ctx *Context) bool {
 	}
 	return true
 }
+
+func fetchNamespaces(ctx context.Context, k8sClient dynamic.Interface) (map[string]string, error) {
```

Comment:
Nitpick: The function naming confused me a little. You're not fetching namspaces but a namespace to org conversion table

Reply:
feel free to reopen if you think about a better name 😅